baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/IllegalSafeLoggingArgumentTest.java

```diff
@@ -861,6 +861,47 @@ public void testInvalidValue_AnnotatedSupertype() {
                 .doTest();
     }
 
+    @Test
+    void testUnsafeInterfaceMethodAndSafeImplementation() {
+        helper().addSourceLines(
+                        "Test.java",
+                        "import com.palantir.logsafe.*;",
+                        "class Test {",
+                        "  interface MyInterface {",
+                        "    @Unsafe",
+                        "    String unsafeMethod();",
+                        "  }",
+                        "  class MyClass implements MyInterface {",
+                        "    @Safe",
+                        "    public String unsafeMethod() {",
+                        "      return \"unsafe\";",
+                        "    }",
+                        "  }",
+                        "}")
+                .doTest();
+    }
+
+    // TODO(#2668): the @Safe interface method should conflict with the @Unsafe class method
```

Comment:
I wasn't sure if this was an omission in IllegalSafeLoggingArgument or in SafeLoggingPropagation, so added to both.

@carterkozak Am I right that this is a situation where we should be failing?